pulpcore/app/serializers/fields.py

```diff
@@ -193,8 +193,12 @@ class BaseURLField(serializers.CharField):
     """
 
     def to_representation(self, value):
-        if settings.CONTENT_HOST:
-            host = settings.CONTENT_HOST
-        else:
-            host = self.context['request'].get_host()
-        return ''.join([host, settings.CONTENT_PATH_PREFIX, value])
+        base_path = value
+        host = settings.CONTENT_HOST or ''
+        prefix = settings.CONTENT_PATH_PREFIX or ''
```

Comment:
This will always be set because the default it's defined in settings.py as '/pulp/content/'. I suggest removing the `or ''` part since it won't be used.

Reply:
There isn't anything preventing a user to setting it to `None` which we should deal with gracefully.